packages/keyring-api/src/rpc.ts

```diff
@@ -178,6 +185,32 @@ export type GetAccountBalancesResponse = Infer<
   typeof GetAccountBalancesResponseStruct
 >;
 
+// ----------------------------------------------------------------------------
+// Resolve account address
+
+export const ResolveAccountAddressRequestStruct = object({
+  ...CommonHeader,
+  method: literal('keyring_resolveAccountAddress'),
+  params: object({
+    scope: CaipChainIdStruct,
```

Comment:
Do we have the scope and is it needed? I wonder if we will have it for off-chain signing requests.

Reply:
From [CAIP-27](https://github.com/ChainAgnostic/CAIPs/blob/main/CAIPs/caip-27.md), we can assume that it's always available:

> **scope** (required) - a valid [CAIP-2](https://chainagnostic.org/CAIPs/caip-2) network identifier, previously authorized by or within a `scopeObject` in the active session